pkg/cache/clusterqueue_test.go

```diff
@@ -367,3 +368,163 @@ func TestClusterQueueUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestClusterQueueUpdateWithAdmissionCheck(t *testing.T) {
+	cq := utiltesting.MakeClusterQueue("cq").
+		AdmissionChecks("check1", "check2", "check3").
+		Obj()
+
+	testcases := []struct {
+		name                      string
+		curStatus                 metrics.ClusterQueueStatus
+		admissionChecks           map[string]AdmissionCheck
+		singleInstanceControllers []string
+		wantStatus                metrics.ClusterQueueStatus
+		wantInactiveReason        string
+	}{
+		{
+			name:       "Pending clusterQueue updated valid AC list",
+			curStatus:  pending,
+			wantStatus: active,
+			admissionChecks: map[string]AdmissionCheck{
+				"check1": {
+					Active:     true,
+					Controller: "controller1",
+				},
+				"check2": {
+					Active:     true,
+					Controller: "controller2",
+				},
+				"check3": {
+					Active:     true,
+					Controller: "controller3",
+				},
+			},
+			wantInactiveReason: "Ready",
```

Comment:
```suggestion
			wantReason: "Ready",
```
It looks weird to say inactiveReason: Ready